### ISSUE TITLE
Contact the Authors 

### ISSUE THREAD

**Issue by turnersr** [shawntan/neural-turing-machines#1]
Have you contacted the authors in order to learn more about the training algorithm used in the paper?

**Comment by jarrelscy** [comment]
I think they said they use RMSProp.

**Comment by shawntan** [comment]
Yes and I've tried RMSProp. I'm looking for more specifics, like initialisation parameters, gradient clipping, etc.

**Comment by jarrelscy** [comment]
Ah yeah, sorry just realised you'd already had that.
